web-client/src/components/WorkForReviewComponent.jsx

```diff
@@ -0,0 +1,394 @@
+                        className={style.form}
+                        noValidate
+                        onSubmit={handleSubmit((data) => {
+                        data.stars = value
+                        alert(JSON.stringify(data))
+                        })}
+                    >
+                        <DialogTitle id='alert-dialog-title' className={style.dialogTitle}>
+                            {'Add your review'}
+                        </DialogTitle>
+                        <DialogContent>
+                            <Box component='fieldset' mb={3} borderColor='transparent' className={style.range}>
+                                <Typography component='legend' className={style.grade}>Grade:</Typography>
+                                <Rating className={style.stars}
+                                    max={3}
+                                    value={value}
+                                    onChange={(event, newValue) => {
+                                        if(newValue !== null)
+                                            setValue(newValue);
+                                    }}
+                                    onChangeActive={(event, newHover) => {
+                                        setHover(newHover);
+                                    }}
+                                />
+                                {value !== null && <Box ml={2} className={style.label}>{labels[hover !== -1 ? hover : value]}</Box>}
+                            </Box>
+                            <p className={style.info}>Review must contain comment or/and file</p>
+                        </DialogContent>
+                        <DialogContent>
+                        <TextField
+                            className={style.textField}
+                            inputRef={register}
+                            required
+                            id="comment-work-for-review"
+                            name="comment"
+                            label="Comment"
+                            autoComplete="comment"
+                            InputLabelProps={{
+                                shrink: true,
+                            }}
+                            InputProps={{
+                                endAdornment: (
+                                    <InputAdornment position="end">
+                                        {counts.comment}/{maxCommentSize}
+                                    </InputAdornment>
+                                ),
+                            }}
+                            multiline
+                            rows={6}
+                            placeholder="Add text"
+                            onChange={handleChange("comment")}
+                            variant="outlined"
+                            error={!!errors.comment}
+                            helperText={errors?.comment?.message}
+                        />
+                        </DialogContent>
```

Comment:
Document is not formatted corectly => `alt + shift + f`

Reply:
ok sry